getting_started/first_3d_game/01.game_setup.rst

```diff
@@ -0,0 +1,164 @@
+.. _doc_first_3d_game_game_area:
+
+Setting up the game area
+========================
+
+In this first part, we're going to set up the game area. Let's get started by
+importing the start assets and setting up the game scene.
+
+We've prepared a Godot project with the 3D models and sounds we'll use for this
+tutorial, linked in the index page. If you haven't done so yet, you can download
+the archive here: `Squash the Creeps assets
+<https://github.com/GDQuest/godot-3d-dodge-the-creeps/releases/tag/1.0.0>`__.
+
+Once you downloaded it, extract the .zip archive on your computer. Open the
+Godot project manager and click the *Import* button.
+
+|image1|
+
+In the import popup, enter the full path to the freshly created directory
+``squash_the_creeps_start/``. You can click the *Browse* button on the right to
+open a file browser and navigate to the ``project.godot`` file the folder
+contains.
+
+|image2|
+
+Click *Import & Edit* to open the project in the editor.
+
+|image3|
+
+The start project contains an icon and two folders: ``art/`` and ``fonts/``.
+There, you will find the art assets and music we'll use in the game.
+
+|image4|
+
+There are two 3D models, ``player.glb`` and ``mob.glb``, some materials that
+belong to these models, and a music track.
+
+Setting up the playable area
+----------------------------
+
+We're going to create our main scene with a plain *Node* as its root. In the
+*Scene* dock, click the *Add Node* button represented by a "+" icon in the
+top-left and double-click on *Node*. Name the node "Main". Alternatively, to add
+a node to the scene, you can press Ctrl a (or Cmd a on MacOS).
```

Comment:
The standard convention seems to be to include a "+" between key presses (so "Ctrl+a" instead of "Ctrl a") to indicate the keys should be pressed simultaneously

Reply:
The keys are missing the keyboard tags, I'll add them. With them you usually don't put the plus sign. Thanks for pointing this out.

Reply:
Thanks, they're missing the keyboard marks too, I'll add them right away.

Reply:
Actually, it seems the kbd styles are a bit broken, they don't work well with the + now

![Screenshot from 2020-11-30 08-51-43](https://user-images.githubusercontent.com/12694995/100624811-72386780-32e9-11eb-8ab2-66a08a346ec3.png)